modules/mono/editor/bindings_generator.h

```diff
@@ -787,7 +787,9 @@ class BindingsGenerator {
 
 	void _initialize();
 
-public:
```

Comment:
Removing this `public:` makes the methods below private, which doesn't seem intentional.

Reply:
Yes, I did that on purpose because the generate_ methods aren't called anywhere outside the bindings generator, they're called from inside the bindings generator - `generate_cs_api` calls the others and calculates paths for them - so it didn't look like they were meant to be public api and just happened to be made public. But if they should be public api, happy to move them back!

Reply:
I don't know if they should be public, but this kind of change would probably be better in a separate PR. I recommend avoiding refactoring or stylistic changes unless they are relevant to the goal of the PR because, otherwise, it may add noise that makes it more difficult to review the PR and would take longer to get it merged.